### PR TITLE
Add initial runtime/sbf/isa.md

### PR DESCRIPTION
**Pull request by riptl**
**Summary**

Relates to #6 

Adds content explaining:
- Instruction encoding
- Register model
- Opcode tables
- Basic static verifier constraints

**Reviewers**

- *Required:* Solana Labs VM team: @jackcmay @Lichtso @alessandrod
- *Optional:* Security researchers

**Organization**

- *Section:* Runtime
- *Status:* Standard, live on mainnet

### CODE REVIEW

File: runtime/sbf/isa.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
+# Sealevel Bytecode Format
+
+Sealevel Bytecode Format (SBF) derives from the eBPF little-endian architecture.
```
Comment (terorie) [inline runtime/sbf/isa.md:3]: Is it "Sealevel Bytecode Format" or "Solana Bytecode Format"??

File: runtime/sbf/isa.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+|  7· |  -  | ldxb | stb | stxb | rsh32  | jsge |  -  | rsh64  |  -   | ldxdw | stdw | stxdw | rsh32  | jsge  |  -  | rsh64  |
+|  8· |  -  |  -   |  -  |  -   | neg32  | call |  -  | neg64  |  -   |   -   |  -   |   -   |   -    | callx |  -  |   -    |
+|  9· |  -  |  -   |  -  |  -   | mod32  | exit |  -  | mod64  |  -   |   -   |  -   |   -   | mod32  |   -   |  -  | mod64  |
+|  A· |  -  |  -   |  -  |  -   | xor32  | jlt  |  -  | xor64  |  -   |   -   |  -   |   -   | xor32  |  jlt  |  -  | xor64  |
+|  B· |  -  |  -   |  -  |  -   | mov32  | jle  |  -  | mov64  |  -   |   -   |  -   |   -   | mov32  |  jle  |  -  | mov64  |
+|  C· |  -  |  -   |  -  |  -   | arsh32 | jslt |  -  | arsh64 |  -   |   -   |  -   |   -   | arsh32 | jslt  |  -  | arsh64 |
+|  D· |  -  |  -   |  -  |  -   |   le   | jsle |  -  |   -    |  -   |   -   |  -   |   -   |   be   | jsle  |  -  |   -    |
+|  E· |  -  |  -   |  -  |  -   | sdiv32 |  -   |  -  | sdiv64 |  -   |   -   |  -   |   -   | sdiv32 |   -   |  -  | sdiv64 |
+|  F· |  -  |  -   |  -  |  -   |   -    |  -   |  -  |   -    |  -   |   -   |  -   |   -   |   -    |   -   |  -  |   -    |
```
Comment (terorie) [inline runtime/sbf/isa.md:85]: should make first row and first column bold

File: runtime/sbf/isa.md (hunk 1 above)
Comment (Lichtso) [reply runtime/sbf/isa.md:3]: "Solana Bytecode Format" see https://github.com/solana-labs/solana/issues/20323

File: runtime/sbf/isa.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+|  6· |  -  | ldxw | stw | stxw | lsh32  | jsgt |  -  | lsh64  |  -   | ldxh  | sth  | stxh  | lsh32  | jsgt  |  -  | lsh64  |
+|  7· |  -  | ldxb | stb | stxb | rsh32  | jsge |  -  | rsh64  |  -   | ldxdw | stdw | stxdw | rsh32  | jsge  |  -  | rsh64  |
+|  8· |  -  |  -   |  -  |  -   | neg32  | call |  -  | neg64  |  -   |   -   |  -   |   -   |   -    | callx |  -  |   -    |
+|  9· |  -  |  -   |  -  |  -   | mod32  | exit |  -  | mod64  |  -   |   -   |  -   |   -   | mod32  |   -   |  -  | mod64  |
+|  A· |  -  |  -   |  -  |  -   | xor32  | jlt  |  -  | xor64  |  -   |   -   |  -   |   -   | xor32  |  jlt  |  -  | xor64  |
+|  B· |  -  |  -   |  -  |  -   | mov32  | jle  |  -  | mov64  |  -   |   -   |  -   |   -   | mov32  |  jle  |  -  | mov64  |
+|  C· |  -  |  -   |  -  |  -   | arsh32 | jslt |  -  | arsh64 |  -   |   -   |  -   |   -   | arsh32 | jslt  |  -  | arsh64 |
+|  D· |  -  |  -   |  -  |  -   |   le   | jsle |  -  |   -    |  -   |   -   |  -   |   -   |   be   | jsle  |  -  |   -    |
+|  E· |  -  |  -   |  -  |  -   | sdiv32 |  -   |  -  | sdiv64 |  -   |   -   |  -   |   -   | sdiv32 |   -   |  -  | sdiv64 |
```
Comment (Lichtso) [inline runtime/sbf/isa.md:84]: Some instructions such as `sdiv` exist in SBFv2 only.

**Pull request closed without merging**